cli/lsp/language_server.rs

```diff
@@ -327,7 +327,7 @@ impl LanguageServer {
       if let Some(lockfile) = cli_options.maybe_lockfile() {
         let lockfile = lockfile.lock();
         if let Err(err) = lockfile.write() {
-          lsp_warn!("Error writing lockfile: {}", err);
+          lsp_warn!("Error writing lockfile: {:#}", err);
```

Comment:
I can't find what `:#` does, is it supposed to be `:?`?

Reply:
It outputs the error with context, which is way better than the standard `{}`. It's buried in the anyhow docs here: https://docs.rs/anyhow/1.0.80/anyhow/struct.Error.html#display-representations